### ISSUE TITLE
Valid v-model and filters

### ISSUE THREAD

**Issue by armano2** [vuejs/eslint-plugin-vue#700]
**Tell us about your environment**	
* **ESLint version:**	^5.2.0
* **eslint-plugin-vue version:**	5.0.0
* **Node version:**	10

**Please show your full configuration:**
test case in project

**What did you do?**	

```vue
<template>
    <input v-model="xx | myfilter">
</template>
```

**What actually happened?**
```
v-model' directives require the attribute value which is valid as LHS.
```
